app/models/errors.rb

```diff
@@ -36,10 +36,10 @@ def satisfied?
     #   that are insufficient.
     ###
     def message
-      super.to_s + ("<ul><li>" + insufficient_items.map do |i|
+      super.to_s + (" " + insufficient_items.map do |i|
         "#{i[:quantity_requested]} #{i[:item_name]} requested, only #{i[:quantity_on_hand]} available." \
         "(Reduce by #{i[:quantity_requested].to_i - i[:quantity_on_hand].to_i})"
-      end.join("</li><li>") + "</li></ul>")
```

Comment:
So while this may not be an ideal refactor, I thought it'd be helpful to get rid of the `<ul>` and `<li>` elements because they were showing up like 

<img width="1176" alt="Screenshot 2023-02-04 at 9 03 25 PM" src="https://user-images.githubusercontent.com/13841769/216797846-4f3f3a4a-a843-4a47-a490-d908a62b115e.png">

With this code change, you get the following behavior

<img width="1161" alt="Screenshot 2023-02-04 at 9 05 16 PM" src="https://user-images.githubusercontent.com/13841769/216797858-e7bffa5b-805e-4b1c-9f1c-624e49643a70.png">